about.html

```diff
@@ -13,51 +13,80 @@
+            </div>
+        </section>
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+        <section class="card-list">
+          <div class="card">
+              <img src="img/about-plan.png" alt="strategy">
 
+              <h2>Strategy</h2>
 
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+              <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
-    Learn More
+              <button>Learn More</button>
+          </div>
 
+          <div class="card"">
+            <img src="img/about-working.png" alt="strategy">
 
+            <h2>How We Work</h2>
 
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+            <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
-    Learn More
+            <button>Learn More</button>
+          </div>
 
+          <div class="card">
+            <img src="img/about-office.png" alt="strategy">
 
+            <h2>Places We Work</h2>
 
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
+            <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+            <button>Learn More</button>
+          </div>
 
-    Learn More
+          <div class="card">
+            <img src="img/about-meeting.png" alt="strategy">
 
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+            <h2>Collaboration</h2>
 
-    Learn More
+            <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
-    Let's Work Together
+            <button>Learn More</button>
+          </div>
+        </section>
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+        <section class="bottom-about">
```

Comment:
Again instead of attaching a new specific class for this button you could have used specificity to style it.